samples/docker/docker-compose.yaml

```diff
@@ -21,18 +21,18 @@ services:
     container_name: azurite
     image: mcr.microsoft.com/azure-storage/azurite
     restart: always
-    ports:
-      - "10001:10001"
-      - "10000:10000"
+    # ports:
```

Comment:
nit: maybe add a comment around explaining to un-comment these if you want connect directly to these services from the host.

Reply:
Done